Jenkinsfile

```diff
@@ -1,3 +1,5 @@
 PalladioPipeline {
     deployUpdatesite 'releng/org.palladiosimulator.experimentautomation.updatesite/target/repository'
+    mavenGoal '-Prelease clean verify'
+    skipDeploy false
```

Comment:
We added these lines for temporary deployment but we should remove them when merging back to master.